var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -64,6 +63,7 @@ class Neuron(Package):
                         '--without-x']
```

Comment:
these changes are already in #369. should that be closed then?

Reply:
Just merged. Will update here to narrow the change